OpenRA.Game/Sound/Sound.cs

```diff
@@ -69,7 +69,8 @@ ISoundSource LoadSound(string filename)
 						return soundEngine.AddSoundSourceFromMemory(rawData, channels, sampleBits, sampleRate);
 				}
 
-				throw new InvalidDataException(filename + " is not a valid sound file!");
```

Comment:
I explained in IRC earlier today what the problem was and how to fix it, if somebody else wants to take a stab at that.